README.md

```diff
@@ -144,10 +144,13 @@ To build the Open XML SDK
 
 If you want to use a command line approach:
 
-1. Go to the directory that contains the solution.
 1. Run `dotnet restore` in the directory.
```

Comment:
Can you keep the step about going to the directory with the solution? The phrase `in the directory` doesn't explain which directory